### ISSUE TITLE
Pylint doesn't work on line breaks due to line length

### ISSUE THREAD

**Issue by Paulchen5** [pylint-dev/pylint#10417]
### Bug description

```python
The following example doesn't work

query_plan = utils.__get_original_query_plan(
    file_content
)  # pylint: disable=protected-access # pyright: ignore[reportPrivateUsage]


this exmaple works for me:

query_plan = utils.__get_original_query_plan(file_content)  # pylint: disable=protected-access # pyright: ignore[reportPrivateUsage]
```

### Configuration

```ini

```

### Command used

```shell
pipx run pylint --ignore=.venv /<absolute-path-to-project>/


But the problem also happens when using the vs code pylint extension
```

### Pylint output

```python
************* Module tests.test_utils
tests/test_utils.py:117:21: W0212: Access to a protected member __get_original_query_plan of a client class (protected-access)
```

### Expected behavior

No output as I disabled it for the line

### Pylint version

```shell
pylint 3.3.7
astroid 3.3.10
Python 3.12.0 (main, Dec 23 2024, 00:59:44) [Clang 16.0.0 (clang-1600.0.26.4)]
```

### OS / Environment

Darwin

### Additional dependencies

```python
click==8.2.1
mypy==1.16.0
mypy_extensions==1.1.0
pathspec==0.12.1
-e git+https://<github-project-path>
typing_extensions==4.14.0
```

**Comment by paulchen5** [comment]
It seems like I should use the following syntax to make it work:

```py
query_plan = (
    utils.__get_original_query_plan(  # pylint: disable=protected-access # pyright: ignore[reportPrivateUsage]
        file_content
    )
)
```

**Comment by Pierre-Sassoulas** [comment]
There's also `# pylint: disable-next=...` on the previous line. This is documented in message control.
